templates/_base_embed.html

```diff
@@ -84,13 +84,16 @@
   {% block content %}{% endblock %}
 
   {# Metric is need by sw registration and page code. #}
-  <script>{% inline_file "/static/js/metrics.min.js" %}</script>
+  <script>{% inline_file "/static/js/metric.min.js" %}</script>
```

Comment:
I was wondering why you renamed the file from metrics to metric?

Reply:
This file is a Metric class. I thought it makes more sense to be singular.